CMakeLists.txt

```diff
@@ -53,4 +54,5 @@ target_include_directories(roboteam_utils
         PRIVATE src )
 
 target_link_libraries(roboteam_utils
-        PUBLIC roboteam_proto)
+        PUBLIC roboteam_proto
+        gtest)
```

Comment:
Very minor, but I tend to like the ) on the next line since it causes less merge conflicts as lines can more easily be added.